api/setup.py

```diff
@@ -19,7 +19,9 @@
 
 REQUIRES = ["connexion[swagger-ui]==2.2.0",
             "python_dateutil==2.6.0",
-            "setuptools>=21.0.0"
+            "setuptools>=21.0.0",
```

Comment:
Please include these dependencies in the requirements.txt file too

Reply:
I have redesigned the requirements.txt file to include only wazuh, api and dependencies from module.